store/tikv/2pc.go

```diff
@@ -538,6 +538,19 @@ func (c *twoPhaseCommitter) prewriteSingleBatch(bo *Backoffer, batch batchKeys)
 			if err1 != nil {
 				return errors.Trace(err1)
 			}
+			if !c.isPessimistic && lock.TTL >= uint64(config.MinPessimisticTTL/time.Millisecond) {
```

Comment:
Large transaction's TTL may be greater than MinPessimisticTTL. The comment below isn't accurate.